README.md

```diff
@@ -26,6 +26,14 @@ will automatically see the updates.
 
 ## The website
 
+### 0. Add previous year to `archives.yaml`
+
+Currently this needs to be run manually once per year, in the new year.
+
+```bash
+./year2yaml $(date -v-1y +%Y)
```

Comment:
I have version 9.0 of GNU date and it barfs on this:
```
date -v-1y +%Y
date: invalid option -- 'v'
Try 'date --help' for more information.
```
Is the command right? Does it need a later version of `date`?

Reply:
I did this to get it to work on macOS, but I'll find something more portable.

Reply:
This slightly more verbose version works for me:

`date --date='last year' +%Y`

Don't know how portable it is, though. Have no Macs here to test on.

Reply:
>  date --date='last year' +%Y
> date: illegal option -- -
> usage: date [-jnRu] [-I[date|hours|minutes|seconds]] [-f input_fmt]
>             [-r filename|seconds] [-v[+|-]val[y|m|w|d|H|M|S]]
>             [[[[mm]dd]HH]MM[[cc]yy][.SS] | new_date] [+output_fmt]

Reply:
Having said that, it's just a Perl script that we run once/year. Ideally the script itself could just "do the right thing".